docs/proposals/PROPOSAL_E.md

```diff
@@ -179,6 +187,7 @@ For registries that do not support the `referrers` API, a tag MUST be pushed for
 - `<ref>`: the digest from the `refers` field (limit of 64 characters)
 - `<hash>`: the digest of this artifact (limit of 16 characters)
 - `<type>`: type of artifact for filtering (limit of 5 characters)
+  - The type field is based on a mapping from `artifactType` to a short string for the tag.
```

Comment:
trying to consolidate the world of new artifacts (in human-readable form) to 5 characters doesn't feel very scalable.

Reply:
Agreed. Hopefully it won't be long to get support for the API so that reliance on these is minimized.